Add clarification about instant reroll effects (#1173)
Heegu-sama/Homm3BG: sections/combat.tex

You may only use \textbf{one Spell per Combat Round}, unlike being able to play any number of other cards.
Ongoing \svg{ongoing} and \svg{activation} activation effects can be used only \textbf{when activating one of your units and before it attacks}.
Ongoing effects\index{Ongoing Effects} last until end of Combat or if the effect on the card is used~up.\par

> Instant \svg{instant} cards may be played \textbf{at any time} except between rolling the Attack Die and resolving damage unless otherwise stated. \textbf{Reroll effects} affecting the Attack Die are an exception to this and may be played after the die is rolled.

Reviewer: ```suggestion
Instant \svg{instant} cards may be played \textbf{at any time} except between rolling the Attack Die and resolving damage unless otherwise stated.
\textbf{Reroll effects} affecting the Attack Die are an exception to this and may be played after the die is rolled.
```
Let's stick to the "one sentence per line" principle.

> Instant \svg{instant} cards may be played \textbf{at any time} except between rolling the Attack Die and resolving damage unless otherwise stated. \textbf{Reroll effects} affecting the Attack Die count as such an exception and may be played after the die is rolled.

Reviewer 2: ```suggestion
Instant \svg{instant} cards may be played \textbf{at any time} except between rolling the Attack Die and resolving damage unless otherwise stated. \textbf{Reroll effects} affecting the Attack Die are an exception to this and may be played after the die is rolled.
```

I understand the sentence, but I stumble on "Attack Die count", thinking it's about number of dice, and have to reread it. I think this variant is more clear
